pages/incidents/incidentList.js

```diff
@@ -1,27 +1,40 @@
 import styled from "styled-components";
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 import { ListContext } from "../../context/listContext";
 import ListEntry from "../../components/listEntry";
+
 function IncidentList() {
   const { incidents, setIncidents } = useContext(ListContext);
+  console.log(incidents);
+  function handleRemove(id) {
+    console.log(id);
+    const newList = incidents.filter((listEntry) => listEntry.id !== id);
+
+    setIncidents(newList);
+  }
+
   return (
     <Container>
       <Headline>Deine Erlebnisse</Headline>
       <List>
         {incidents?.map((listEntry) => (
```

Comment:
du könntest listEntry hier destructuren, bin mir aber gerade nicht sicher, ob das hier tatsächlich weniger Schreibarbeit wäre...

```suggestion
        {incidents?.map((listEntry) => {
                const {id, time, location, offender, incident} = listEntry;
                return (
                <ListEntry>
                ...
                </ListEntry>
                )
        }
```